README.md

```diff
@@ -4,25 +4,30 @@ Crawler to collect newspaper articles for analysis
 Requirements
 ------------
 * Docker
+* Docker-Compose
 
 And that's it. Docker will get any additional requirements and setup your development or testing environment.
 
```

Comment:
Docker purely provides container functionality and the interface to start/stop/connect to containers. Most environments based on Docker use several containers, and docker-compose provides an interface and config options to start a multiple container environment on a single machine. 

See the .yaml files in the deployment folder, for the docker-compose configs.

As we may be moving to more serious deployments in the future, we may have to look into docker-swarm, or kubernetes, which handle docker containers over mutiple machines.